best-practices/README.md

```diff
@@ -219,6 +219,12 @@ HTML
 CSS
 ---
 
+* Document the project's CSS architecture in the README, including things
+  such as:
+  * Organization of stylesheet directories and Sass partials
+  * Selector naming
+  * Browser support
+  * Code linting
```

Comment:
Can you be more specific? As in "What tools you are using for code linting" or "The library that is being used for code linting"?